Can't read/write WebP on Windows because of UNIX permission check

Hi, I've been using scrimage in my toy project and it comes error when i tried to use scrimage-webp on window.

it looks like scrimage-webp add executable permission when loading cwebp, dwebp binaries
which is not supported by Window. 

it works fine without adding permission 


![image](https://user-images.githubusercontent.com/50672087/106431866-38363e00-64b1-11eb-846d-90f5b193e1b9.png)

```
Process finished with exit code -1
... 
io.kotest.engine.spec.runners.SingleInstanceSpecRunner$execute$2$invokeSuspend$$inlined$invoke$lambda$2.invokeSuspend(SingleInstanceSpecRunner.kt:56)
	at io.kotest.engine.spec.runners.SingleInstanceSpecRunner$execute$2$invokeSuspend$$inlined$invoke$lambda$2.invoke(SingleInstanceSpecRunner.kt)
	at io.kotest.engine.launchers.SequentialTestLauncher$launch$$inlined$forEach$lambda$1$1.invokeSuspend(SequentialTestLauncher.kt:22)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ExceptionInInitializerError
	at com.sksamuel.scrimage.webp.WebpImageReader.<init>(WebpImageReader.java:11)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:380)
	... 83 more
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "chmod": CreateProcess error=2, 지정된 파일을 찾을 수 없습니다
	at com.sksamuel.scrimage.webp.DWebpHandler.<clinit>(DWebpHandler.java:20)
	... 90 more
Caused by: java.io.IOException: Cannot run program "chmod": CreateProcess error=2, 지정된 파일을 찾을 수 없습니다 (can't find the file you looking for)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at com.sksamuel.scrimage.webp.WebpHandler.setExecutable(WebpHandler.java:33)
	at com.sksamuel.scrimage.webp.WebpHandler.installBinary(WebpHandler.java:23)
	at com.sksamuel.scrimage.webp.DWebpHandler.installDWebp(DWebpHandler.java:32)
	at com.sksamuel.scrimage.webp.DWebpHandler.<clinit>(DWebpHandler.java:18)
	... 90 more
Caused by: java.io.IOException: CreateProcess error=2, 지정된 파일을 찾을 수 없습니다 (can't find the file you looking for)
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:444)
	at java.lang.ProcessImpl.start(ProcessImpl.java:139)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 94 more
```

--- a/buildSrc/src/main/kotlin/Libs.kt
+++ b/buildSrc/src/main/kotlin/Libs.kt
@@ -3,7 +3,8 @@ object Libs {
    const val kotlinVersion = "1.4.21"
 
    const val org = "com.sksamuel.scrimage"
-   const val CommonsIoVersion = "2.6"
+   const val CommonsIoVersion   = "2.6"
+   const val CommonsLangVersion = "3.11"
 
    object TwelveMonkeys {
       private const val Version = "3.6"
@@ -40,6 +41,7 @@ object Libs {
    }
 
    object Commons {
-      const val io = "commons-io:commons-io:$CommonsIoVersion"
+      const val io   = "commons-io:commons-io:$CommonsIoVersion"
+      const val lang = "org.apache.commons:commons-lang3:$CommonsLangVersion"
    }
 }

--- a/scrimage-webp/build.gradle.kts
+++ b/scrimage-webp/build.gradle.kts
@@ -5,6 +5,7 @@ plugins {
 
 dependencies {
    api(project(":scrimage-core"))
+   implementation(Libs.Commons.lang)
    testImplementation(Libs.Kotest.junit5)
    testImplementation(Libs.Kotest.assertions)
 }

--- a/scrimage-webp/src/main/java/com/sksamuel/scrimage/webp/WebpHandler.java
+++ b/scrimage-webp/src/main/java/com/sksamuel/scrimage/webp/WebpHandler.java
@@ -1,5 +1,7 @@
 package com.sksamuel.scrimage.webp;
 
+import org.apache.commons.lang3.SystemUtils;
+
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.file.Files;
@@ -20,7 +22,10 @@ abstract class WebpHandler {
          if (in != null) {
             Files.copy(in, output, StandardCopyOption.REPLACE_EXISTING);
             in.close();
-            setExecutable(output);
+
+            if(!SystemUtils.IS_OS_WINDOWS) {
+               setExecutable(output);
+            }
             return;
          }
       }

